### PR TITLE
small PR for merge_artifact

### PR DESCRIPTION
**Pull request by latran**
Changing strategy="merge" to strategy="last"

### CODE REVIEW

**Review by JacobCallahan — APPROVED** [review]
ACK. So the intent of this is to allow Tower administrators to use jobs that a user doesn't have permission to view in the workflow execution process. As long as they make the last job (that presents results) viewable by the user, broker will still process them correctly.

**Comment by mshriver** [comment]
@JacobCallahan Since we're using py3.8 features, we need to turn 3.6 and 3.7 of for travis.  @latran want to include that in this PR, its just removing two lines from `.travis.yml`?

**Comment by latran** [comment]
@mshriver, @JacobCallahan Ok done.

File: .travis.yml [hunk 1]
```diff
@@ -1,8 +1,6 @@
 sudo: true
 language: python
 python:
-    - "3.6"
```
Comment (mshriver) [inline .travis.yml:4]: Thanks for including this!

Comment (latran) [reply .travis.yml:4]: @mshriver ?.  Are you saying thanks for including the removal? If so, English is weird.

Comment (mshriver) [reply .travis.yml:4]: yes and yes, it sure is weird.

**Review by mshriver — APPROVED** [review]

**Pull request merged**